numpy/ma/tests/test_extras.py

```diff
@@ -1123,6 +1123,16 @@ def test_2d_with_missing(self):
                              x.shape[0] / frac))
 
 
+    def test_shapes(self):
+        #test 1-d and 2-d inputs
+        data = np.asarray(np.random.rand(5,3))
+        y = np.asarray(np.random.rand(2,3))
+        assert_almost_equal(cov(data,y=y), np.cov(data,y=y))
+
+        data = np.asarray(np.random.rand(2,))
+        y = np.asarray(np.random.rand(2,))
+        assert_almost_equal(cov(data,y=y), np.cov(data,y=y))
```

Comment:
These tests don't belong here - this file is for testing the maskedarray implementation.

There is already a test for 1d data at `np.lib.tests.test_function_base.TestCov.test_1D_rowvar`, so this isn't needed at all